# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../slim/slim-1.3.2-X11-conf.patch
# Copyright (C) 2011 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- slim-1.3.2/slim.conf.orig	2010-07-08 07:04:10.000000000 +0200
+++ slim-1.3.2/slim.conf	2011-03-22 20:14:50.908072045 +0100
@@ -1,17 +1,17 @@
 # Path, X server and arguments (if needed)
 # Note: -xauth $authfile is automatically appended
-default_path        /bin:/usr/bin:/usr/local/bin
-default_xserver     /usr/bin/X
+default_path        /bin:/usr/bin:/usr/local/bin:/usr/X11/bin
+default_xserver     /usr/X11/bin/X
 #xserver_arguments   -dpi 75
 
 # Commands for halt, login, etc.
 halt_cmd            /sbin/shutdown -h now
 reboot_cmd          /sbin/shutdown -r now
-console_cmd         /usr/bin/xterm -C -fg white -bg black +sb -T "Console login" -e /bin/sh -c "/bin/cat /etc/issue; exec /bin/login"
+console_cmd         /usr/X11/bin/xterm -C -fg white -bg black +sb -T "Console login" -e /bin/sh -c "/bin/cat /etc/issue; exec /bin/login"
 #suspend_cmd        /usr/sbin/suspend
 
 # Full path to the xauth binary
-xauth_path         /usr/bin/xauth 
+xauth_path         /usr/X11/bin/xauth 
 
 # Xauth file for server
 authfile           /var/run/slim.auth
@@ -51,7 +51,7 @@
 # The current chosen session name is replaced in the login_cmd
 # above, so your login command can handle different sessions.
 # see the xinitrc.sample file shipped with slim sources
-sessions            xfce4,icewm,wmaker,blackbox
+sessions            openbox,xfce4,icewm,wmaker,blackbox
 
 # Executed when pressing F11 (requires imagemagick)
 screenshot_cmd      import -window root /slim.png
